baseimages/components/rhel_7_9/python.yml

```diff
@@ -0,0 +1,50 @@
+---
+name: baseimage_rhel_7_9_python
+description: rhel 7.9 base image component for installing python
+schemaVersion: 1.0
+parameters:
+  - Version:
+      type: string
+      default: "0.0.1"
+      description: Component version (update this each time the file changes)
+  - Platform:
+      type: string
+      default: "Linux"
+      description: Platform.
+phases:
+  - name: build
+    steps:
+      - name: InstallPython3-9
+        action: ExecuteBash
+        inputs:
+          commands:
+            - |
+              set -e
+              cd /tmp
+
+              py_version=3.9.6
```

Comment:
nice, however, it would be cleaner, if you have defined it under the `parameters`, then when it comes to upgrading the version, it's all at the top, where you have to change it